dev-docs/source/Testing/VSI/VSITesting.rst

```diff
@@ -0,0 +1,61 @@
+.. _vsi_testing:
+
+VSI Testing
+=============
+
+.. contents::
+   :local:
+
+
+*Preparation*
+
+- Get the scripts from the Mantid repository `this directory <https://github.com/mantidproject/mantid/tree/master/scripts/vsi>`_.
+- Make sure that you have the Mantid system test data. You can do this by building a SystemTestData target of Mantid.
+- Put the system test data directory in your Mantid user directories
+
+
+**Time required 20-30 minutes**
+
+--------------
+
+#. Load the script SXD_NaCl.py in MantidPlot. Run it.
+#. This should load a workspace called QLab. Right-click this workspace and choose 'Show VSI simple interface'.
```

Comment:
Sorry to be pedantic, but the menu option is actually "Show VATES simple interface"